source/Application.cpp

```diff
@@ -23,18 +24,26 @@ namespace FileShare::GUI {
 
     void Application::init()
     {
-        // tgui::Theme::setDefault("assets/themes/Black.txt");
+        auto &themeManager = ThemeManager::getInstance();
 
         tgui::ScrollablePanel::Ptr content = tgui::ScrollablePanel::create();
         content->setAutoLayout(tgui::AutoLayout::Fill);
+        if (themeManager.getCurrentTheme() == "dark") {
+            content->getRenderer()->setBackgroundColor(tgui::Color(35, 35, 35));
+        }
```

Comment:
Hum, why do we have to set this here on `init` ?

Are theses things gonna update properly when we switch themes ?
If so, they should be setup properly without us having to do anything ?